src/components/GameView/MoveChoiceCard.vue

```diff
@@ -85,8 +85,15 @@ export default {
   cursor: pointer;
 }
 .move-choice-card {
-  border: 2px solid #fff;
+  background-color: rgba(var(--v-theme-surface-1));
+  color: rgba(var(--v-theme-surface-2)) !important;
```

Comment:
` !important` smells here but it didn't work without it and I don't think it's a huge deal

Reply:
I'd urge you to reconsider adding this, but it's fine for now. Longer term I think you need to just increase the specificity. I'm also surprised since these component styles are scoped. Could also add an ID for the unique dialogs which would supercede it, but that also doesn't feel right.

It's possible you're using this to override the children when maybe the children rules need to be adjusted instead.

Reply:
I'll do some further thinkering to see if I can get the specificity right. I also initially thought scoped styles would do it, but I think that because they ultimately just boil down to classes with hashed names that the scoped styles actually _don't_ increase the specificity, they just prevent style leaks into other files. I think your suggestion of an id selector makes sense; I'll try that first

Reply:
Created https://github.com/cuttle-cards/cuttle/issues/398 to track fixing the `!important` usages